app/main.ipynb

```diff
@@ -192,18 +237,35 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 20,
-   "outputs": [],
    "source": [
-    "# write your code here"
+    "A = np.array([\n",
+    "    [1, 6],\n",
+    "    [2, 8],\n",
+    "    [3, 11],\n",
+    "    [3, 10],\n",
+    "    [1, 7]\n",
+    "])\n",
+    "\n",
+    "sum_first_column = np.sum(A[:, 0])\n",
+    "print(\"sum_first_column:\", sum_first_column)"
```

Comment:
The task requires finding the mean for each column, but the code currently calculates the sum of the first column. Use `np.mean(A, axis=0)` to find the mean for each column.